homeworks/src/homework4/part2/Robot.java

```diff
@@ -0,0 +1,11 @@
+package homework4.part2;
+
+public class Robot extends Participant{
+
+    public Robot (int runLimit, int jumpLimit, String name) {
+        setRunLimit(runLimit);
+        setJumpLimit(jumpLimit);
+        setName(name);
+    }
```

Comment:
In your implementation no sense to have classes Robot, Cat, Human - no specific. Make sense to have only class Participant with a constructors